scripts/vim-patch.sh

```diff
@@ -180,8 +180,12 @@ preprocess_patch() {
   local file="$1"
   local nvim="nvim -u NORC -i NONE --headless"
 
-  # Remove *.proto, Make*, gui_*, some if_*
-  local na_src='proto\|Make*\|gui_*\|if_lua\|if_mzsch\|if_olepp\|if_ole\|if_perl\|if_py\|if_ruby\|if_tcl\|if_xcmdsrv'
+  # Remove Filelist, README
+  local na_files='Filelist\|README.*'
+  2>/dev/null $nvim --cmd 'set dir=/tmp' +'g@^diff --git a/\<\%('${na_files}'\)\>@norm! d/\v(^diff)|%$
' +w +q "$file"
+
+  # Remove *.proto, Make*, gui_*, beval.*, some if_*, gvim, libvterm, tee, VisVim, xpm, xxd
+  local na_src='auto\|configure.*\|GvimExt\|libvterm\|proto\|tee\|VisVim\|xpm\|xxd\|Make*\|beval.*\|gui_*\|if_lua\|if_mzsch\|if_olepp\|if_ole\|if_perl\|if_py\|if_ruby\|if_tcl\|if_xcmdsrv'
```

Comment:
I don't check `src/auto` or the configure scripts. I figure that the CMake configs and the supported OS are different such that the configure scripts are not portable.